.github/workflows/tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Pytest
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [2.7, 3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest six
+    - name: Pytest
+      run: |
+        pytest -vv
```

Comment:
oh this is cool -- so now it has a CI that can run all the forking tests?

Reply:
Yeah, you can check CI jobs [here](https://github.com/skhortiuk/ufork/actions)